src/pybind/mgr/dashboard/frontend/src/app/shared/components/carbon-notification-panel/carbon-notification-panel.component.html

```diff
@@ -0,0 +1,89 @@
+<div class="cds--side-panel" [class.active]="isOpen">
+  <header class="cds--side-panel__header">
+    <h2 class="cds--side-panel__title">
+      <svg cdsIcon="notification" size="20"></svg>
+      Notifications
+    </h2>
+    <button class="cds--side-panel__close" (click)="close()">
```

Comment:
We should be using carbon buttons , did we check that ? if yes, then why vanilla button ?